src/components/App.js

```diff
@@ -12,27 +15,45 @@ export const styles = {
   },
 };
 
-function App({ classes }) {
-  return (
-    <React.Fragment>
-      <TopBar />
-      <Grid container>
-        <Grid item md={9} sm={12} xs={12} className={classes.calendar}>
-          <CalendarViewContainer />
-        </Grid>
+export const mapDispatchToProps = dispatch => ({
+  fetchSchools: () => {
+    dispatch(fetchSchools('4720'));
+  },
+});
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.classes = props.classes;
+    this.fetchSchools = props.fetchSchools;
```

Comment:
You rarely ever want to set state using props. Here, there's no need to set any state - you don't even need a constructor. Just use the props themselves in the render & lifecycle method.

Reply:
This will also fix your coverage issue.